Updating BCs Documentation (#250)
shannon-lab/zapdos: doc/content/source/bcs/SakiyamaSecondaryElectronBC.md

# SakiyamaSecondaryElectronBC

!syntax description /BCs/SakiyamaSecondaryElectronBC

## Overview

> `SakiyamaSecondaryElectronBC` is an electron mean energy density of secondary electrons induced by ion flux outflow boundary condition.
> `SakiyamaSecondaryElectronBC` assumes the electrostatic approximation for the electric field.

Reviewer: ```suggestion
`SakiyamaSecondaryElectronBC` accounts for the mean electron energy density of secondary electrons induced by an ion flux outflow boundary condition.
`SakiyamaSecondaryElectronBC` assumes the electrostatic approximation for the electric field.
```

# SakiyamaSecondaryElectronBC

> !syntax description /BCs/SakiyamaSecondaryElectronBC

Reviewer: Class description requires another space

## Overview

`SakiyamaSecondaryElectronBC` is an electron mean energy density of secondary electrons induced by ion flux outflow boundary condition.
`SakiyamaSecondaryElectronBC` assumes the electrostatic approximation for the electric field.

> The ion induce secondary electron mean energy density outflow is defined as

Reviewer: ```suggestion
The ion induced secondary electron mean energy density outflow is defined as
```

> \begin{equation}
> a =
> \begin{cases}
> 1, & \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} > 0\\
> 0, & \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} \leq 0\\
> \end{cases} \\[10pt]
> \Gamma_{e} \cdot \textbf{n} = \gamma \left[a \ \mu_{j} \ \text{-} \nabla (V) \cdot \textbf{n} \ n_{j} \right]
> \end{equation}

Reviewer: ```suggestion
\begin{equation}
a =
\begin{cases}
1, & \mu_{j} \left( \text{-} \nabla (V) \right) \cdot \textbf{n} > 0\\
0, & \mu_{j} \left( \text{-} \nabla (V) \right) \cdot \textbf{n} \leq 0\\
\end{cases} \\[10pt]
\Gamma_{e} \cdot \textbf{n} = \gamma \left[a \ \mu_{j} \left( \text{-} \nabla (V) \right) \cdot \textbf{n} \ n_{j} \right]
\end{equation}
```

> Where $\Gamma$ is the outflow normal to the boundary, $\textbf{n}$ is the normal of the boundary,
> $\mu_{j}$ is the mobility coefficient, $n_{j}$ is the ion density, $\gamma$ is the secondary electron coefficient, and $V$ is the potential. $a$ is defined such that the outflow is only defined when the drift velocity is direct towards the wall and zero otherwise. When converting the density to log form and applying a scaling factor of the mesh, the strong form for `SakiyamaSecondaryElectronBC` is defined as

Reviewer: ```suggestion
Where $\Gamma_e \cdot \textbf{n}$ is the flux normal to the boundary, $\textbf{n}$ is the normal vector of the boundary,
$\mu_{j}$ is the mobility coefficient, $n_{j}$ is the ion density, $\gamma$ is the secondary electron coefficient, and $V$ is the electric potential. $a$ is defined such that the outflow is only non-zero when the drift velocity is directed towards the wall and zero otherwise. When converting the density to log form and applying a scaling factor of the mesh, the strong form for `SakiyamaSecondaryElectronBC` is defined as
```

> \begin{equation}
> \Gamma_{e} \cdot \textbf{n} = \gamma \left[a \ \mu_{j} \ \text{-} \nabla (V / l_{c}) \cdot \textbf{n} \ \exp(N_{j}) \right]
> \end{equation}

Reviewer: ```suggestion
\begin{equation}
\Gamma_{e} \cdot \textbf{n} = \gamma \left[a \ \mu_{j} \left( \text{-} \nabla (V / l_{c}) \right) \cdot \textbf{n} \ \exp(N_{j}) \right]
\end{equation}
```

> Where $N_{j}$ is the molar density of the specie in log form and $l_{c}$ is the scaling factor of the mesh.

Reviewer: ```suggestion
Where $N_{j}$ is the molar density of the species in log form and $l_{c}$ is the scaling factor of the mesh.
```

## Example Input File Syntax

> An example of how to use `SakiyamaEnergySecondaryElectronBC` can be found in the
> test file `2D_RF_Plasma_actions.i`.

Reviewer: ```suggestion
```
